### ISSUE TITLE
Inverse transit returns something that looks like a normal transit?

### ISSUE THREAD

**Issue by eblur** [lkreidberg/batman#33]
Hi there, I have a student fitting a light curve and our optimizer started returning negative values for the planet radius. After checking it out, I see that the code allows for negative values in the case of an "inverse transit". But for some reason it still returns something that looks like a regular transit. I thought you should know that this is happening.

Here's the test we ran:

```
import numpy as np
import matplotlib.pyplot as plt

import batman

print("Batman version", batman.__version__)

params = batman.TransitParams()
params.t0 = 0.                       #time of inferior conjunction
params.per = 1.                      #orbital period
params.rp = -0.1                      #planet radius (in units of stellar radii)
params.a = 15.                       #semi-major axis (in units of stellar radii)
params.inc = 87.                     #orbital inclination (in degrees)
params.ecc = 0.                      #eccentricity
params.w = 90.                       #longitude of periastron (in degrees)
params.u = [0.1, 0.3]                #limb darkening coefficients [u1, u2]
params.limb_dark = "quadratic"       #limb darkening model

t = np.linspace(-0.05, 0.05, 100)

m = batman.TransitModel(params, t)    #initializes model

print("Transit model radius =", m.rp)
print("Is this an inverse transit?", m.inverse)

flux = m.light_curve(params)          #calculates light curve

plt.plot(t, flux)
plt.xlabel("Time from central transit")
plt.ylabel("Relative flux")
plt.show()
```

**Comment by eblur** [comment]
The problem is that, when handling the case of rp < 0, the original code changes the value `params.rp`. So when I run `batman.TransitModel.light_curve` with the same `batman.TransitParam` object, it switches back to a normal transit.

I have modified the code in PR #47 so that the original parameters object does not get modified, `TransitModel.rp = np.abs(params.rp)` and the transit is always calculated with the absolute value of `params.rp`

**Comment by lkreidberg** [comment]
this is fixed in batman-package version 2.4.8; thanks @eblur
